qiskit/visualization/circuit_visualization.py

```diff
@@ -615,6 +616,7 @@ def _matplotlib_circuit_drawer(circuit,
                                justify=None,
                                idle_wires=True,
                                with_layout=True,
+                               initial_state=True,
```

Comment:
```suggestion
                               initial_state=False,
```

Sorry for the misunderstanding in https://github.com/Qiskit/qiskit-terra/issues/4293#issuecomment-622654114. By consistency I mean with `draw`. MPL is not been tested now.

Reply:
Done. Also set it to False for matplotlib.py in the __init__ in MatplotlibDrawer.